projects/samples/contests/robocup/protos/RobocupGoal.proto

```diff
@@ -32,8 +32,13 @@ PROTO RobocupGoal [
   post_radius = 0.05 -- Diameter/width/depth must be between 0.08-0.12
   post_spacing = D + 2 * post_radius
   post_height = GH + post_radius
+<<<<<<< HEAD
```

Comment:
All these lines are wrong and should not be committed.
These are the files used by the merge tool.

Reply:
Yes, I know, this a draft PR which I am fixing now.